src/components/Editor/SessionList.vue

```diff
@@ -132,6 +138,14 @@ export default {
 			return this.participantsWithoutCurrent.slice(0, 3)
 		},
 	},
+	methods: {
+		applyFocus() {
+			this.$refs.form.focus()
+		},
+		revertFocus() {
+			this.$editor.chain().focus().run()
```

Comment:
Should this rather go back to focus on the button after closing?

Reply:
I don't know. But I am open for ideas.